tests/functional/pages/surveys/checkbox_other/0/checkbox-other-summary.page.js

```diff
@@ -0,0 +1,12 @@
+import QuestionPage from '../../question.page'
```

Comment:
We can drop the checkbox-other- from the filename as it's in a checkbox_other folder?
Why the '0' folder?